core/store/src/trie/mod.rs

```diff
@@ -94,15 +97,15 @@ const TRIE_COSTS: TrieCosts = TrieCosts { byte_of_key: 2, byte_of_value: 1, node
 
 #[derive(Clone, Hash)]
 enum NodeHandle {
-    InMemory(StorageHandle),
+    // InMemory(StorageHandle),
```

Comment:
Lol, where was NodeHandle::InMemory() being used? In case it doesn't exist any more, could we just simplify this structure from enum to `struct NodeHandle(CryptoHash)`?

Reply:
We can, later. It should've died completely with `TrieNodeWithSize`, but as I mentioned it is used somewhere else in codebase.